tests/test_integrations_locally.py

```diff
@@ -23,7 +23,7 @@ def model_Training_DemoNotebook(self):
 
 class PySyftNotebooks(unittest.TestCase):
 
-    def paillier_HEExampleNotebook(self):
+    def paillier_HE_example_notebook(self):
```

Comment:
I'm leaving HE capitalized since it is an abbreviation. 
share any other thoughts on this